### DISCUSSION TITLE
How to handle HTML header meta tags?

### DISCUSSION THREAD

**Discussion by arnonrdp** [intlify/vue-i18n#917] [Q&A]
I would like to optimize for SEO and for that I intend to translate the HTML header meta tags.

I don't just mean the `meta title`, but also the `meta description` and `meta keywords`.

**Comment by kazupon** [comment]
I'm going to release the package called `vue-i18n-routing`.
That package has a composition APIs that provides SEO-friendly features called `useHead`.

When you are combining that with `@vueuse/head`, I believe you can make your wishes realized.

**Reply by arnonrdp** [reply]
I didn't know about [`@vueuse/head`](https://github.com/vueuse/head). Now I'm wondering if I apply the translation there, will it not work?
